nexum_core/benches/README.md

```
@@ -0,0 +1,215 @@
+5. **Document Performance Targets**: Include expected performance ranges
+6. **Test Edge Cases**: Include boundary conditions and error cases
+
+### Example Benchmark Structure
+```rust
+fn my_new_benchmark(c: &mut Criterion) {
+    let mut group = c.benchmark_group("my_feature");
+    
+    for size in [100, 1000, 10000].iter() {
+        group.throughput(Throughput::Elements(*size as u64));
+        
+        group.bench_with_input(
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)